rest/src/test/scala/io/udash/rest/RestTestApi.scala

```diff
@@ -124,6 +124,7 @@ trait RestTestApi {
   ): Future[String]
 
```

Comment:
missing test case for default group name when not aggregated in `describedGroup`